packages/client-sdk-web/src/config/configuration.ts

```diff
@@ -0,0 +1,46 @@
+import {MomentoLoggerFactory} from '../common/config/logging';
+
+export interface ConfigurationProps {
```

Comment:
so did you decide you do need this in order for the abstract classes to work?

Reply:
this is independent of the abstract class, it can be extended without needing these props. This is just here so we can pass the logger to the web-sdk. I left the other attributes commented out that I felt like we might use it the future.

Reply:
okay but the logger/config object isn't actually required to get it to compiler for now?

Reply:
the web-sdk is using the logger, so it is required for compiling rn https://github.com/momentohq/client-sdk-nodejs/pull/369/files#diff-7a6be8180d030dba51f316bc567e3546e1924e68f5cc8bcc8ac98097e14c96f6R43

Reply:
got it.